docs/storefront/graphql/best-practices.mdx

```diff
@@ -0,0 +1,34 @@
+# Best Practices for Using GraphQL APIs to Ensure Platform Stability & Reliability
+
+GraphQL APIs provide merchants with powerful data retrieval capabilities, but without proper management, they can strain platform stability and performance. To maintain a seamless experience for all users, merchants should adhere to the following best practices:
+
+1. **Implement Caching for Better Performance** 
+   * Reduce API response times by caching frequently requested data.
+   * Minimize redundant processing to improve scalability and handle more users efficiently.
+   * Ensure cache invalidation mechanisms update data appropriately to prevent stale responses.
+2. **Use a Trusted Proxy for Stability & Security**
```

Comment:
I think we need here a link to trusted proxy implementation and the emphasis should be more on the fact that merchants need to use trusted proxy and propage true client IP to GQL in case they use proxy